README.md

```diff
@@ -40,6 +40,9 @@ This project is tested with [BrowserStack](https://www.browserstack.com/home).
 |---|---|---|
 |Linux|[![Build Status](https://github.com/Ascend/onnxruntime/actions/workflows/build-and-test.yaml/badge.svg)](https://github.com/Ascend/onnxruntime/actions/workflows/build-and-test.yaml)||
 
+## Releases
```

Comment:
nit: maybe rename section from Releases to Upcoming Release, unless we want to add information of current and past releases link (https://github.com/microsoft/onnxruntime/releases).

Reply:
Good point! Added that link.